### DISCUSSION TITLE
Support questions

### DISCUSSION THREAD

**Discussion by tommls** [olahallengren/sql-server-maintenance-solution#259] [Questions & Answers]
I have two SQL servers that are not doing backups properly.
Both are SQL 2012, one has specific databases that don't back up, I get SQL severity 016 errors even though I can see *.bak files in the directory, the other one I can see *.bak files but hours later I see numerous 016 error messages.
1) Where do I find the 'application log' for backups??
2) Is it possible to download older versions of the maintenance script?? I think this all started after upgrading the script...
3) Is Github the official support request site??
Thank you, Tom

**Comment by olahallengren** [comment]
1. MaintenanceSolution.sql is creating the jobs with output - files. By default the output - files are located in the SQL Server error log directory. The output - files contain all the details about the errors. I would be very interested in what the error message is.

2. My website has the latest version of the scripts. GitHub has all versions since I started using GitHub.

3. The support channels are GitHub, Stack Exchange (tag ola-hallengren) and e-mail.
